config/localConfig.template.php

```diff
@@ -47,21 +47,47 @@
 
 $debug = false;
 
+/* Set the Environment
+ * Valid values are 'ENV_PROD', 'ENV_DEV' (default), or 'ENV_TEST'
+ *
+ * ENV_DEV and ENV_TEST behaviors:
+ *  - set display errors to 1
+ *  - add test reports to 'View Old Reports' page in UDOIT
+ *  - No Canvas token validation
+ *  - Returns "test-token" when getting refresh token for Canvas
+ *  - Course Scanner doesn't scan, simply contains already loaded test reports
+ */
+$UDOIT_ENV = ENV_DEV; // default
```

Comment:
Sorry I didn't notice this before, but it should really be `ENV_PROD` instead.  While in a perfect world it would be set to DEV for institutions to do some testing during implementation, I believe it would result in confusion.